.github/workflows/main.yml

```diff
@@ -0,0 +1,27 @@
+name: Continuous Integration
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # Make sure the actual branch is checked out when running on pull requests
+          ref: ${{ github.head_ref }}
+          # This is important to fetch the changes to the previous commit
+          fetch-depth: 0
+
+      - name: Prettify code
+        uses: creyD/prettier_action@v3.3 # https://github.com/creyD/prettier_action
+        with:
+          prettier_options: --write .
```

Comment:
You can change the prettier options here to change which files it runs on, or in the `.prettierrc.json`/`.prettierignore` files.